### PR TITLE
Keep SSH tunnel open via periodic ping

### PR DESCRIPTION
**Pull request by CCLDArjun**
Healthcheck tunnels time out if there is no data on the connection. New container to ping it every 5 seconds so the connection isn't lost

### CODE REVIEW

**Review by akshtshrma24 — APPROVED** [review]
Looks good to me

File: tunup/check_tunnel_status.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import logging
+from time import sleep, gmtime
+import requests
+import sys
+
+t = int(sys.argv[1])
```
Comment (evanugarte) [inline tunup/check_tunnel_status.py:6]: sys was good for when we were building it but now that were actually merging it this should be `argparse`

```py
import argparse

parser = argparse.ArgumentParser()
parser.add_argument(
  "--request_interval_seconds",
  type=int,
  default=5,
  help="Interval to query flightradar24"
)
parser.add_argument(
  "--hosts",
  type=str,
  help="comma separated values for hosts to send requests to \
    i.e. http://host.docker.internal:11000/api/health-check,http://host.docker.internal:14000/healthcheck/printer"
)

args = parser.parse_args()

print(args.request_interval_seconds)
print(args.hosts)

hosts_list = [host.strip for host in args.hosts.split(',')]

# iterate through hosts list and u can ping it easy
```

File: tunup/READMe [hunk 2]
```diff
@@ -0,0 +1 @@
+keep tunnel from quasar to core-v4 up by periodically sending http request
```
Comment (evanugarte) [inline tunup/READMe:1]: rename this to README.md

File: docker-compose.yml [hunk 3]
```diff
@@ -7,6 +7,13 @@ services:
     volumes:
       - '/data/db:/data/db' 
     command: 'mongod'
+  tunup:
```
Comment (evanugarte) [inline docker-compose.yml:10]: how about `keep-ssh-tunnel-open`?

File: tunup/check_tunnel_status.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  type=str,
+  required=True,
+  help="comma separated values for hosts to send requests to \
+    i.e. http://host.docker.internal:11000/api/health-check,http://host.docker.internal:14000/healthcheck/printer"
+)
+
+args = parser.parse_args()
+
+hosts_list = [host.strip() for host in args.hosts.split(',')]
+print(hosts_list)
```
Comment (evanugarte) [inline tunup/check_tunnel_status.py:25]:
```suggestion
logging.info(f"using hosts {hosts_list}")
```

File: tunup/check_tunnel_status.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+# iterate through hosts list and u can ping it easy
+
+t = int(args.request_interval_seconds)
```
Comment (evanugarte) [inline tunup/check_tunnel_status.py:29]: i believe this is already an integer, u can just use it on line 49
```suggestion
```

File: tunup/check_tunnel_status.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+logging.Formatter.converter = gmtime
+logging.basicConfig(
+    format="%(asctime)s.%(msecs)03dZ %(levelname)s:%(name)s:%(message)s",
+    datefmt="%Y-%m-%dT%H:%M:%S",
+    level=logging.INFO,
+)
+
+logging.info("Starting tunnel routine...")
+def ping(host):
+    print(host)
```
Comment (evanugarte) [inline tunup/check_tunnel_status.py:40]:
```suggestion
```

File: tunup/check_tunnel_status.py [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    try:
+        req = requests.get(host)
+        logging.info(f"Success connecting to {host}: %s", req)
+    except Exception as e:
+        logging.error(f"Could not reach {host}: %s", e)
+
+while True:
+    [ping(host) for host in hosts_list]
+    sleep(t)
```
Comment (evanugarte) [inline tunup/check_tunnel_status.py:49]:
```suggestion
    sleep(args.request_interval_seconds)
```

**Review by evanugarte — APPROVED** [review]

**Pull request merged**